pandas/io/formats/format.py

```diff
@@ -574,7 +586,26 @@ def __init__(
             )
         self.na_rep = na_rep
         self.decimal = decimal
-        self.col_space = col_space
+        if col_space is None:
+            self.col_space = {}
+        elif isinstance(col_space, (int, str)):
+            self.col_space = {"": col_space}
```

Comment:
Minor but you can just create the dict inline using a comprehension on one line instead of doing this then calling update on the next

Reply:
@WillAyd The ```""``` (empty) key  is for the row id column - this is from the old implementation. Couldn't think of how to turn this into a oneliner?